networkx/algorithms/bipartite/generators.py

```diff
@@ -524,7 +544,7 @@ def bipartite_gnmk_random_graph(n, m, k, seed=None, directed=False):
         
     Examples
     --------
-    G = nx.bipartite_gnmk_random_graph(10,20,50)
+    G = nx.gnmk_random_graph(10,20,50)
```

Comment:
The example here should import the bipartite package first:

```
from networkx.algorithms import bipartite
G = bipartite.gnmk_random_graph(10,20,50)
```
Given that the doctest does not give an error, are we sure that we are not importing the generators to the main namespace?